pkg/atom/atom_test.go

```diff
@@ -1,23 +1,86 @@
-package atom_test
+package atom
```

Comment:
the issue here, @heronrs, is that I can't init an struct from the atom package because some properties there are private and I can't access inside the _test pkg. For example: 

```
a := &atom.Client{
		Atom: fa,
		k8s:  c,    <-
	}
```

I can change the `k8s` to be public but I think this is even worse.

Reply:
Any chance to use the `New` function?
https://github.com/convox/convox/blob/master/pkg/atom/atom.go#L42
If no, that's ok. I guess it's not properly injecting all dependencies

Reply:
I don't know how, @heronrs. Because the `config`, the param for the `New` function uses the `exec` command. 
```
data, err := exec.Command("kubectl", "config", "view", "--raw").CombinedOutput()
```

Is there a way to mock this?

Reply:
Yeah I see your point, that's cool, let's just keep as is.